rt.yml

```diff
@@ -52,6 +55,9 @@ components:
         - "Contains raw information about barcodes on the input image"
         - "Contains license"
         - "Contains encrypted ResultContainerList"
+        - "Servers for storing the data reading results from the RFID-chip in a form of a list of the logically separated data groups (raw data)"
+        - "Servers for storing the data reading results from the RFID-chip in a form of a list of the logically separated data groups (binary data)"
+        - "Servers for storing the results of data reading in a form of a list of objects of the original binary representation of the graphics in memory of the RFID-chip"
```

Comment:
Needs rewriting
"Servers"? Make Contains as everywhere else
I cannot understand the description at all this way